docs/MigrationGuide.md

```diff
@@ -2,6 +2,14 @@
 
 To see the biggest differences please consult the [changelog](Changelog.md).
 
+## In version 1.4.0 (Not released yet)
+
+### Code API changes
+
+* [#2983](https://github.com/TouK/nussknacker/pull/2983) Extract security api module (Permission, GlobalPermission)
+  * `pl.touk.nussknacker.ui.security.api.Permission` (security module) moved to `pl.touk.nussknacker.security.api.Permission` (extension-api module)
```

Comment:
we've created APIs mostly without changing packages, if we change package here maybe lets put it in right package, do we indroduce the extension package @mproch wdyt ?

Reply:
I think security is not currently considered as "api", so it may be good time to move stuff around. Or try to be backward compatible via depracted class/types in current package?